frontend_app_thread/src/css/index.styl

```diff
@@ -20,16 +20,21 @@
   .timeline
     margin 0
     width 100%
+    height calc(100% - 40px)
```

Comment:
Again a small explanation of this `40px` could be useful.

Reply:
fixed